governance/xc_admin/packages/xc_admin_frontend/components/tabs/General.tsx

```diff
@@ -0,0 +1,730 @@
+                  SECURITY_MULTISIG[getMultisigCluster(cluster)],
+                  1
+                ),
+                productAccount: productAccountKey,
+              })
+              .instruction()
+          )
+          // deterministically generate price account key
+          const priceAccountKey = await PublicKey.createWithSeed(
+            OPS_KEY,
+            'price:' + symbol,
+            pythProgramClient.programId
+          )
+          // create add price account instruction
+          instructions.push(
+            await pythProgramClient.methods
+              .addPrice(newChanges.priceAccounts[0].expo, 1)
+              .accounts({
+                fundingAccount: squads?.getAuthorityPDA(
+                  SECURITY_MULTISIG[getMultisigCluster(cluster)],
+                  1
+                ),
+                productAccount: productAccountKey,
+                priceAccount: priceAccountKey,
+              })
+              .instruction()
+          )
+
+          // create add publisher instruction if there are any publishers
+          if (newChanges.priceAccounts[0].publishers.length > 0) {
+            newChanges.priceAccounts[0].publishers.forEach(
+              (publisherKey: string) => {
+                pythProgramClient.methods
+                  .addPublisher(new PublicKey(publisherKey))
+                  .accounts({
+                    fundingAccount: squads?.getAuthorityPDA(
+                      SECURITY_MULTISIG[getMultisigCluster(cluster)],
+                      1
+                    ),
+                    priceAccount: priceAccountKey,
+                  })
+                  .instruction()
+                  .then((instruction) => instructions.push(instruction))
+              }
+            )
+          }
+
+          // create set min publisher instruction if there are any publishers
+          if (newChanges.priceAccounts[0].minPub) {
+            instructions.push(
+              await pythProgramClient.methods
+                .setMinPub(newChanges.priceAccounts[0].minPub, [0, 0, 0])
+                .accounts({
+                  priceAccount: priceAccountKey,
+                  fundingAccount: squads?.getAuthorityPDA(
+                    SECURITY_MULTISIG[getMultisigCluster(cluster)],
+                    1
+                  ),
+                })
+                .instruction()
+            )
+          }
+        } else {
+          // check if metadata has changed, or minPub has changed, or publishers have changed
+          // check if metadata has changed
+          // check if there are any new metadata by comparing prev and new values
+          if (
+            JSON.stringify(prev.metadata) !==
+            JSON.stringify(newChanges.metadata)
+          ) {
+            // create update product account instruction
+            instructions.push(
+              await pythProgramClient.methods
+                .updProduct(newChanges.metadata)
```

Comment:
You need to add the symbol field here I found this the hard way. Don't forget that symbol lives here as well so you need to do something like:
`updProduct({...newChanges.metadata, symbol: symbol})`